Lesson7/Task1/Shape.java

```diff
@@ -0,0 +1,7 @@
+package Lesson7.Task1;
+
+public abstract class Shape {
+    public abstract double getArea();
+    public abstract double getPerimeter();
+    public abstract String toString();
+}
```

Comment:
ту стрінг краще не вказувати він від object класа наслідується 
```suggestion
    public abstract double getArea();
    public abstract double getPerimeter();
}
```